### ISSUE TITLE
Import-Export drawers 

### ISSUE THREAD

**Issue by heberjeur** [TeamNewPipe/NewPipe#1666]
- [ ] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [ ] I checked if the issue/feature exists in the latest version.
- [ ] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

When I import my newpipe zip data my drawers configuration not restored
Is there any menu to do that ?
Plus, I suggest to show logo channels inside of the actually icon

**Comment by TaranQ** [comment]
I am new to GitHub and to Newpipe. 
I tried Newpipe and love it and i wanted to test the import export database so when updating I don't lose all my stuff. But importing a freshly exported database just clears all my history, playlists and subscriptions?
I'm on OnePlus 6 Android 8.1 OxygenOS 5.1.11

**Comment by theScrabi** [comment]
Yes it overrides your current playlist and restores the one backed up.

**Comment by TaranQ** [comment]
I Backup (export)  NP with a playlist and subscriptions. 
Settings - content - database export.
If I then import the file that was created during the above export my playlist and subscriptions are gone? meaning NP is like freshly installed looks like just all data is being deleted instead of restored.

**Comment by theScrabi** [comment]
This is weird, and you say this only occurs to the drawer?

**Comment by theScrabi** [comment]
I can not follow, the drawer layout may not change? But the front page layout is customizable since v0.14.0, however I could restore that layout.

**Comment by TaranQ** [comment]
I mean that a want to be able to make a backup of my playlists and subscriptions in case I need to re-install my OS or if I am switching to a new phone. But if I try to test this it turns out importing a backup doesn't restore my playlists or subscriptions. Instead it clears all data.

**Comment by theScrabi** [comment]
That is weird. Maybe your backup is broken. I can not confiem this, foe me everything works.

**Comment by TaranQ** [comment]
I have tried multiple times now with no success.
I can see the zip file that is being created. It contains newpipe.db ans newpipe.settings and the Filesize is not zero so I guess export is ok.
BTW I am on OnePlus 6 Android 8.1
